src/baguetteBox.js

```diff
@@ -178,39 +173,33 @@
             var gallery = [];
             selectorData.galleries.push(gallery);
 
-            // Get all gallery images and save them in imagesMap with custom options
-            var galleryID = imagesMap.length;
-            imagesMap.push(tagsNodeList);
-
             [].forEach.call(tagsNodeList, function(imageElement, imageIndex) {
                 var imageElementClickHandler = function(event) {
                     /* jshint -W030 */
                     event.preventDefault ? event.preventDefault() : event.returnValue = false;
-                    prepareOverlay(galleryID, userOptions);
+                    prepareOverlay(gallery, userOptions);
                     showOverlay(imageIndex);
                 };
                 var imageItem = {
                     eventHandler: imageElementClickHandler,
                     imageElement: imageElement
                 };
-                imagedEventHandlers[galleryID + '_' + imageElement] = imageElementClickHandler;
                 bind(imageElement, 'click', imageElementClickHandler);
                 gallery.push(imageItem);
             });
         });
-        console.log(data);
     }
 
     function unbindImageClickListeners() {
-        galleries.forEach(function(gallery) {
-            [].forEach.call(gallery, function() {
-                var galleryID = imagesMap.length - 1;
-                [].forEach.call(imagesMap[galleryID], function(imageElement) {
-                    unbind(imageElement, 'click', imagedEventHandlers[galleryID + '_' + imageElement]);
-                });
-                imagesMap.pop();
-            });
-        });
+        // galleries.forEach(function(gallery) {
```

Comment:
That is why git rebase exists. You should rebase and cleanup/squash any related commits.